doc/howto/instances_snapshots.md

```diff
@@ -0,0 +1,67 @@
+(instances-snapshots)=
+# How to create instance snapshots
+
+You can back up your instance by creating a snapshot of the instance.
+
+A snapshot saves the state of the instance volume at a specific time, which makes it easy to restore the instance to a previous state.
+It is stored in the same storage pool as the instance volume itself.
+
+## Create a snapshot
+
+Use the following command to create a snapshot of an instance:
+
+    lxc snapshot <instance_name> [<snapshot name>]
+
+% Include content from [storage_backup_volume.md](storage_backup_volume.md)
+```{include} storage_backup_volume.md
+    :start-after: <!-- Include start create snapshot options -->
+    :end-before: <!-- Include end create snapshot options -->
+```
+
+To capture not only the data included in the instance volume but also the running state of the instance, add the `--stateful` flag.
```

Comment:
Not a blocker, but due to the state of CRIU this is unlikely to work for containers.
Do we have a section we can embed to make a disclaimer/notice that any "stateful" operation will likely fail for containers at the moment?